src/libplctag/Tag.cs

```diff
@@ -10,6 +12,8 @@ namespace libplctag
     public sealed class Tag : IDisposable, ITag
     {
 
+        private const int asyncStatusPollInterval = 2;
```

Comment:
It's a style thing, but I'm used to constants being named in all caps -> ASYNC_STATUS_POLL_INTERVAL

Reply:
Yep no probs - updated.